src/probnum/quad/_bayesquad.py

```diff
@@ -54,46 +59,50 @@ def bayesquad(
     Parameters
     ----------
     fun :
-        Function to be integrated.
-    input_dim:
+        Function to be integrated. It needs to accept a shape=(n_eval, input_dim) ``np.ndarray`` and return a
+        shape=(n_eval,) ``np.ndarray``
```

Comment:
```suggestion
        shape=(n_eval,) ``np.ndarray``.
```

Reply:
If I ever learn to terminate sentences with full stops :sweat_smile: